src/main/java/net/sf/jabref/exporter/FieldFormatterCleanups.java

```diff
@@ -85,7 +85,7 @@ public int hashCode() {
         return Objects.hash(actions, enabled);
     }
 
-    private static List<FieldFormatterCleanup> parse(String formatterString) {
+    public static List<FieldFormatterCleanup> parse(String formatterString) {
```

Comment:
The this-Call? It is nothing more than overloading, only that in constructors you have to use this...
I think the boolean parameter is indicating  if the selected CleanUp is active or not...
At least there is a function: `public List<String> getAsStringList() {`
which apparently uses the value